auth/src/test/java/com/firebase/ui/auth/testhelpers/TestConstants.java

```diff
@@ -18,6 +18,7 @@
 
 public class TestConstants {
     public static final String EMAIL = "test@example.com";
+    public static final String PHONE = "+14347565353";
```

Comment:
Lol, is this a real phone number? We should probably go with `+15555555555` or something similar.